include/osquery/sql.h

```diff
@@ -16,12 +16,24 @@
 
 #include <osquery/database.h>
 #include <osquery/flags.h>
+#include <osquery/registry.h>
 #include <osquery/tables.h>
 
 namespace osquery {
 
 DECLARE_int32(value_max);
 
+class only_movable {
```

Comment:
NB: Usually this is what we mean when we inherit `boost::noncopyable`, but to avoid refactoring even more, we'll keep this local.